### DISCUSSION TITLE
A new release version with ForceStopAsyncSend in Fluentd driver

### DISCUSSION THREAD

**Discussion by aabadia-dlocal** [moby/moby#43842] [Q&A] [answered]
### Description

I would need a new version that includes this [commit](https://github.com/moby/moby/commit/5cbc08ce57a54f8532516812efb0e5297c06c2b7) in 20.10.x release. It is vital to refresh connections due to changes at the DNS level and to balance TCP connections.

[The flag ForceStopAsyncSend was added to fluent logger lib in v1.9.0](https://github.com/moby/moby/commit/5cbc08ce57a54f8532516812efb0e5297c06c2b7
)

**Comment by cpuguy83** [comment, marked as answer]
I don't think we'll be backporting this to 20.10.
However this commit is in 23.0 which is due to be released soon.

You can also package up fluentd as a plugin (I know some people already have) which gives more control over what version and other things are available.
